db/pokemon_data.rb

```diff
@@ -0,0 +1,11 @@
+def get_pokemon_data
```

Comment:
this cool that you followed our model for seeding, but this isn't necessary! you can just shove everything in the seed file if you wanted.